Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -103,302 +105,325 @@ def run
+                                                              silent:                   args.quiet?,
+                                                              ignore_non_pypi_packages: true
+
+            update_matching_version_resources! formula,
+                                               version: new_formula_version.to_s
+          end
+
+          pr_message = "Created with `brew bump-formula-pr`."
+          if resources_checked.nil? && formula.resources.any? do |resource|
+            resource.livecheck.formula != :parent && !resource.name.start_with?("homebrew-")
+          end
+            pr_message += <<~EOS
 
-            - [ ] `resource` blocks have been checked for updates.
-          EOS
-        end
 
-        if new_url =~ %r{^https://github\.com/([\w-]+)/([\w-]+)/archive/refs/tags/(v?[.0-9]+)\.tar\.}
-          owner = Regexp.last_match(1)
-          repo = Regexp.last_match(2)
-          tag = Regexp.last_match(3)
-          github_release_data = begin
-            GitHub::API.open_rest("#{GitHub::API_URL}/repos/#{owner}/#{repo}/releases/tags/#{tag}")
-          rescue GitHub::API::HTTPNotFoundError
-            # If this is a 404: we can't do anything.
-            nil
+              - [ ] `resource` blocks have been checked for updates.
+            EOS
           end
 
-          if github_release_data.present?
-            pre = "pre" if github_release_data["prerelease"].present?
-            pr_message += <<~XML
-              <details>
-                <summary>#{pre}release notes</summary>
-                <pre>#{github_release_data["body"]}</pre>
-              </details>
-            XML
+          if new_url =~ %r{^https://github\.com/([\w-]+)/([\w-]+)/archive/refs/tags/(v?[.0-9]+)\.tar\.}
+            owner = Regexp.last_match(1)
+            repo = Regexp.last_match(2)
+            tag = Regexp.last_match(3)
+            github_release_data = begin
+              GitHub::API.open_rest("#{GitHub::API_URL}/repos/#{owner}/#{repo}/releases/tags/#{tag}")
+            rescue GitHub::API::HTTPNotFoundError
+              # If this is a 404: we can't do anything.
+              nil
+            end
+
+            if github_release_data.present?
+              pre = "pre" if github_release_data["prerelease"].present?
+              pr_message + <<~XML
+                <details>
+                  <summary>#{pre}release notes</summary>
+                  <pre>#{github_release_data["body"]}</pre>
+                </details>
+              XML
+            end
           end
+
+          {
+            sourcefile_path:  formula.path,
+            old_contents:,
+            commit_message:   "#{formula.name} #{new_formula_version}",
+            additional_files: alias_rename,
+          }
         end
 
+        tap = pr_formula.tap
+
+        pr_message = args.message
+
+        # run_audit(formula, alias_rename, old_contents)
```

Comment:
```suggestion
        run_audit(formula, alias_rename, old_contents)
```
Again, just a reminder – and for context to others looking at this PR, these lines have been commented out just to make testing easy.